platform/android/android_native_app_glue.h

```diff
@@ -50,7 +50,7 @@ extern "C" {
  *
  * 2/ android_main() receives a pointer to a valid "android_app" structure
  *    that contains references to other important objects, e.g. the
- *    ANativeActivity obejct instance the application is running in.
```

Comment:
This file (`platform/android/android_native_app_glue.h`) and `platform/android/cpu-features.c` come directly from Google, so it's better not to modify them in our repo. I should move them to `thirdparty/` eventually.